src/Previewer.js

```diff
@@ -97,20 +118,31 @@ export default function Previewer({
       return;
     }
 
+    dbus_proxy.DarkMode = style_manager.dark;
+    
```

Comment:
this doesn't work - dbus_proxy is null at this point in time.

I'll proceed with merging anyway - appreciate if you can send a follow up PR :pray: